css/layout.css

```diff
@@ -1,7 +1,3 @@
-.hidden {
-  display: none !important;
-}
-
```

Comment:
But why?

Reply:
Whoops! not sure how that happened. Will fix